src/print/index.ts

```diff
@@ -229,18 +229,21 @@ export function print(path: FastPath, options: ParserOptions, print: PrintFn): D
 
             if (isAttributeShorthand && options.svelteAllowShorthand) {
                 return concat([line, '{', node.name, '}']);
-						} else {
-							const def: Doc[] = [line, node.name];
-							if (node.value !== true) {
-									def.push('=');
-									const quotes = !hasLoneMustacheTag || options.svelteStrictMode;
-
-									quotes && def.push('"');
-									def.push(...path.map(childPath => childPath.call(print), 'value'));
-									quotes && def.push('"');
-							}
-							return concat(def);
-						}
+			} else {
```

Comment:
it looks like there are tabs on this line and spaces on the other lines

Reply:
Yep, fixed. Thanks.